perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -840,9 +863,26 @@ def Prepare(benchmark_spec):
   # Setup common test tools required on the client VM
   vms[0].Install('sysbench05plus')
 
+
+  vms[0].mysql_svc_oltp_tables_count = FLAGS.mysql_svc_oltp_tables_count
```

Comment:
We traditionally store state on the benchmark_spec itself, not the vm object. I would suggest doing that here, since these flag values are not vm-specific.